script/elevate-cpanel.PL

```diff
@@ -961,6 +962,10 @@ sub run_stage_2 ($self) {
     $self->ssystem_and_die(qw{/scripts/update-packages});
     $self->ssystem_and_die(qw{/usr/bin/yum -y update});
 
+    # This needs to happen before the database upgrade because
+    # it will remove the packages we want to detect
+    $self->run_component_once( 'PackageRestore' => 'pre_leapp' );
```

Comment:
Sorry to ask this but will this happen before or after the auto-upgrade of MySQL?

Reply:
Before the auto-upgrade.  The auto-upgrade will cause net-snmp to be removed; so I had to detect the package beforehand.